.buildkite/scripts/install_deps.sh

```diff
@@ -6,7 +6,7 @@ set -euo pipefail
 
 platform_type="$(uname)"
 hw_type="$(uname -m)"
-platform_type_lowercase="${platform_type,,}"
```

Comment:
Not supported in the bash version available in Macos ARM VM. It requires bash >4.0